ngx-tools/src/input-has-changed/input-has-changed.spec.ts

```diff
@@ -6,7 +6,7 @@ import {
 import { inputHasChanged } from './input-has-changed';
 
 
-describe(`inputHasChanged`, () => {
+describe(`inputHasChanged with string input`, () => {
```

Comment:
I would slightly adjust these `describe`s so it's a little easier in the future to target a test.

I've been trying to keep one primary containing `describe` for each component/service/etc so that the entire component can be targeted.

Something like:

```
describe(`inputHasChanged`)
  describe(`with string`)
    test...
  describe(`with object`)
    test...
```